modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/release-notes-breaking-change-search/ddm-template.ftl

```diff
@@ -0,0 +1,306 @@
+<style>
+	a.link-container:hover, a.link-container.hover {
+		background-color: var(--color-action-primary-hover-10, #EDF3FE);
+		border: 0.0625rem solid var(--link-hover-color, #0053F0);
+		color: var(--link-hover-color, #0053F0);
+		text-decoration: none;
+	}
+
+	.collapse-container {
+		position: relative;
+	}
+
+	.collapse-container .tooltip {
+		margin-top: -100px;
+		transform: translateX(-40%);
+		width: 140px;
```

Comment:
Can't it be `rem` here? To be consistent with the rest of the file

Reply:
The class was removed and not in use.